content-vs/dotnet-template-azure-iot-edge-module/CSharp/module.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema-version": "0.0.1",
+    "description": "",
+    "image": {
+        "repository": "<repository>",
```

Comment:
Why is a copy of this file needed to be maintained here?

Reply:
arm32 platfrom is not included in this file